### PR TITLE
Add instructions for setting up Jutut

### PR DESCRIPTION
**Pull request by etanttila**
# Description

**What?**

Add instructions on how a course can start using Jutut. Provide general instructions for creating feedback exercises, making the necessary changes in conf.py, as well as adding Jutut to the side menu.

**Why?**

There haven't been clear instructions for teachers before, and many teachers have been unsure how to start using Jutut

**How?**

Adding a chapter to Aplus manual

# Testing

**Remember to add or update unit tests for new features and changes.**

* How to [test your changes in A-plus](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations)
* How to [test accessibility](https://wiki.aalto.fi/display/EDIT/How+to+check+the+accessibility+of+pull+requests)

**What type of test did you run?**

- [ ] Accessibility test using the [WAVE](https://wave.webaim.org/extension/) extension.
- [ ] Django unit tests.
- [ ] Selenium tests.
- [ ] Other test. *(Add a description below)*
- [x] Manual testing.

Tested the A+ chapter manually.
Someone should check if following these instructions is enough for setting up Jutut in the production environment

**Did you test the changes in**

- [ ] Chrome
- [x] Firefox
- [ ] This pull request cannot be tested in the browser.

**Think of what is affected by these changes and could become broken**

# Translation

- [ ] Did you modify or add new strings in the user interface? ([Read about how to create translation](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations))

# Programming style

- [x] Did you follow our [style guides](https://apluslms.github.io/contribute/styleguides/)?
- [ ] Did you use Python type hinting in all functions that you added or edited? ([type hints](https://docs.python.org/3/library/typing.html) for function parameters and return values)

# Have you updated the README or other relevant documentation?

- [ ] documents inside the doc directory.
- [ ] README.md.
- [x] Aplus Manual.
- [ ] Other documentation (mention below which documentation).

# Is it Done?

- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

*Clean up your git commit history before submitting the pull request!*

### CODE REVIEW

File: jutut/setting_up.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
+Setting up Jutut
+================
+
+Creating feedback questionnaires
+---------------------------------
```
Comment (ihalaij1) [inline jutut/setting_up.rst]: There is one too many dashes.

**Review by ihalaij1 — CHANGES_REQUESTED** [review]
I'm not sure why, but I get this error when I delete the previous `_build` directory and then try to build:
```
...
reading sources... [ 30%] languages/languages

Sphinx error:
Missing translation for up and key question
make: *** [Makefile:33: html] Error 2
```

**Comment by etanttila** [comment]
> I'm not sure why, but I get this error when I delete the previous `_build` directory and then try to build:
> 
> ```
> ...
> reading sources... [ 30%] languages/languages
> 
> Sphinx error:
> Missing translation for up and key question
> make: *** [Makefile:33: html] Error 2
> ```

Yep, I have the same problem (also when making changes to the file), I always first have to compile a version without this chapter and only after that build a version with this chapter. I have no idea where this issue comes from and how it could be fixed.

**Review by ihalaij1 — APPROVED** [review]
For some reason, the problem was the name of the file `setting_up.rst`...
Changed it to `set_up_jutut.rst` and everything works now.

**Pull request merged**